cmd/construct.go

```diff
@@ -0,0 +1,129 @@
+package cmd
```

Comment:
The file name is ambiguous, i have added a file called readme.go to the main branch please compare and move the contents to that file

Reply:
Sure thing, will do it when have time.

Reply:
Thanks for the changes.